sros2_cmake/sros2_cmakeConfig.cmake.in

```diff
@@ -1,9 +0,0 @@
-# Compute paths
-
-set(DEFAULT_KEYSTORE keys)
-set(ros2_security_helperBASE_DIR "${CMAKE_CURRENT_LIST_DIR}/../../..")
-set(DEFAULT_SECURE_FOLDER "${ros2_security_helperBASE_DIR}/ros2_security")
```

Comment:
@ross-desmond this logic doesn't seem to be used anywhere. Is it fine to remove it ? or is the intent to point the default keystore to this `ros2_security` folder?

Reply:
@ross-desmond I'll wait for an answer from you on this before I merge this, thanks!

Reply:
Exactly, the point of this logic is to set up default locations for the keystore and security folder. This will create a ros2_security folder under `ros2_ws/install/ros2_security` if none is specified.

Reply:
thanks @ross-desmond  for the answer.
To clarify the question, the part currently unclear to me is: where is this variable used ?

Currently if I don't specify any argument (keystore or root directory), the artifacts are generated under `build/PACKAGE_NAME/keys`, and no `ros2_security` directory is created in the install space.

Is the expected behavior that the default keystore be located at: `ros2_ws/install/ros2_security/keys` ?

Reply:
@ross-desmond @mjcarroll very friendly :bellhop_bell:

Reply:
@ross-desmond Caan you address @mikaelarguedas's question above so we can get this one wrapped up?

Reply:
My mistake, this variable may not be used by the final implementation that went into sros is looks like. It would be nice to have these installed under ros2_ws/install/ros2_security/keys in order to have a default place for keys created in the colcon build step regardless of the package that created it. What are your thoughts?

Reply:
I think it's a good idea :+1: .
This will need to be implemented differently though as this approach is relative to where the config file is located.
@ross-desmond If it's ok with you, I'll open a follow-up ticket (with a suggested approach and additional questions) rather than implementing it in this PR.

@mjcarroll I'm suggesting to merge this as is as it doesn't change current behavior and addresses the original issue